deployment/README.md

```diff
@@ -134,3 +134,98 @@ Run the following commands to apply the **container_file_t** SELinux label to th
 # semanage fcontext -a -t container_file_t "<STORAGE-PATH>(/.*)?"
 # restorecon -Rv <STORAGE-PATH>
 ```
+Add a new section to https://github.com/IBM/Db2/tree/develop/deployment.
+
+
+# How to create a new LDAP user
+## 1. Allocate the LDAP pod
+``` 
+oc get po | grep ldap | cut -d " " -f 1
+```
+Example output:
+```
+db2u-deployment-db2u-ldap-5cf745f7cf-ql6sh
+```
+## 2. Log in to the LDAP pod
+```
+oc rsh <LDAP-pod-name> /bin/bash
```

Comment:
Could we do this in "1" command? Something like `oc rsh $(oc get po | grep ldap | cut -d " " -f 1) /bin/bash`

Reply:
Good idea. Updated.